### ISSUE TITLE
Keeps changing views when in a meeting

### ISSUE THREAD

**Issue by priyanshul-govil** [TobiasPankner/Teams-Auto-Joiner#73]
**Describe the bug**
After joining the meeting, it keeps looping between 3 different views:
1. Calendar tab with the meeting in a mini-window
2. Full-screen meeting
3. Teams tab with meeting minimized to the top-left corner

**To Reproduce**
Not Applicable. Just run using mode `3`.

**Expected behavior**
It stayed in one view mode and not kept changing views.

**Configuration file**
```json
{
  "run_at_time": "",

  "meeting_mode": 3,
  "organisation_num" : 1,
  "random_delay": false,
  "check_interval": 10,

  "auto_leave_after_min": 60,
  "leave_if_last": true,
  "leave_threshold_number": "15",
  "leave_threshold_percentage": "",
  "pause_search": true,

  "headless" : false,
  "mute_audio": false,
  "chrome_type": "msedge",

  "blacklist": [
    {
      "team_name": "Example Team 1",
      "channel_names": [
        "First Channel to blacklist in this team",
        "Second Channel to blacklist"
      ]
    },
     {
      "team_name": "Blacklist whole team",
      "channel_names": [
      ]
    }
  ],
  "blacklist_meeting_re": ""
}
```

**Desktop (please complete the following information):**
 - OS: `Windows 10 Home 20H2`
 - Browser: `Microsoft Edge`
 - Version: `88.0.705.53`

**Additional context**
Terminal log since meeting join, if it helps.

```
[09:18:23] Looking for new meetings
Found meetings: 
        EM-III Lecture  from 9:15 AM to 10:05 AM 1611891840 [Calendar]
Video off
[18380:14000:0129/091826.382:ERROR:audio_rtp_receiver.cc(90)] AudioRtpReceiver::OnSetVolume: No audio channel exists.
Joined meeting: EM-III Lecture  from 9:15 AM to 10:05 AM
[18380:14000:0129/091831.696:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091831.702:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091831.774:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091831.779:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091831.857:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091831.862:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091831.955:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091831.961:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091832.042:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091832.045:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091832.139:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091832.144:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091832.211:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091832.243:ERROR:internal_decoder_factory.cc(59)] Trying to create decoder for unsupported format
[18380:14000:0129/091832.261:ERROR:webrtc_video_engine.cc(3335)] Absent receive stream; ignoring clearing encoded frame sink for ssrc 0
[18380:14000:0129/091832.290:ERROR:webrtc_video_engine.cc(3335)] Absent receive stream; ignoring clearing encoded frame sink for ssrc 0    
[18380:14000:0129/091832.292:ERROR:webrtc_video_engine.cc(3335)] Absent receive stream; ignoring clearing encoded frame sink for ssrc 0    
[18380:14000:0129/091832.293:ERROR:webrtc_video_engine.cc(3335)] Absent receive stream; ignoring clearing encoded frame sink for ssrc 0    
[18380:14000:0129/091832.294:ERROR:webrtc_video_engine.cc(3335)] Absent receive stream; ignoring clearing encoded frame sink for ssrc 0    
[18380:14000:0129/091832.294:ERROR:webrtc_video_engine.cc(3335)] Absent receive stream; ignoring clearing encoded frame sink for ssrc 0    
[18380:14000:0129/091832.295:ERROR:webrtc_video_engine.cc(3335)] Absent receive stream; ignoring clearing encoded frame sink for ssrc 0    

[09:18:50] Looking for new meetings
Meeting search os paused because you are still in a meeting

[09:19:13] Looking for new meetings
Meeting search os paused because you are still in a meeting
```

**Comment by AlexFernando001** [comment]
Same issue here 👎 @TobiasPankner Please help 🥇

**Comment by TobiasPankner** [comment]
Fixed with commit 0a62572
